tensorflow/python/training/saver_test.py

```diff
@@ -696,7 +684,10 @@ def _testPartitionedVariables(self, use_resource):
     var_full_shape = [10, 3]
     # Allows save/restore mechanism to work w/ different slicings.
     var_name = "my_var"
-    saved_path = os.path.join(_TestDir("partitioned_variables"), "ckpt")
+    saved_dir = os.path.join(self.get_temp_dir(), "partitioned_variables")
```

Comment:
unlike supervisor_test, this test file has a lot of test classes.
So even when we do that, it will still have to be repeated 15 times.

One possible solution might be to create a new test base class in this file that implements the method, and base all others on top of that.
Do you think it is worth it?